node/packages/parser/src/ethereum/index.ts

```diff
@@ -21,15 +22,18 @@ export interface TransactionParserArgs {
   rpcUrl: string
 }
 
-export type Parser = thor.Parser | uniV2.Parser | zrx.Parser
+export interface GenericParser {
```

Comment:
Nice! We can add a generic tx type to support parsers across all chains as well and hoist to `parser/src/index.ts` as well.

Reply:
Isn't that what `StandardTx` already is, or do you mean something else?

Reply:
Right now all of the input payload tx types are blockbook transactions, but this won't always be the case. `GenericParser<T> { parse: (tx: T) }` so we can specific what kind of generic parser based on backend endpoints

Reply:
Addressed in https://github.com/shapeshift/unchained/pull/292